src/main/resources/frontend/src/views/AdminView/SubmissionsView.vue

```diff
@@ -41,6 +41,14 @@ const resetPage = async () => {
   loadSubmissionsToTable(submissionsData);
 }
 
+const refreshSubmissions = async () => {
+  if (allSubmissionsLoaded) {
```

Comment:
This seems reasonable. Ideally, we would only refresh the _CHANGED_ submissions, but that requires an infrastructure more robust than we currently have in place.

Reply:
Yeah that's not currently possible, nor does it seem like something worth putting effort into